server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -825,8 +825,18 @@ private Optional<FormTag> addressCorrectionEnabledToggle(
     boolean questionIsUsedInPredicate =
         programDefinition.isQuestionUsedInPredicate(questionDefinition.getId());
 
+    boolean addressCorrectionEnabledQuestionAlreadyExists =
+        blockDefinition.hasAddressCorrectionEnabledOnDifferentQuestion(questionDefinition.getId());
+
     String toolTipText =
-        "Enabling address correction will check the resident's address to ensure it is accurate.";
+        "Enabling 'address correction' will check the resident's address to ensure it is accurate.";
```

Comment:
why add single quotes here?

Reply:
This was what was in the copy, and seemed to be the only piece that was missing before.